security/fma-incident-response-improvements.md

```diff
@@ -0,0 +1,254 @@
+- Failing to call `OptimismPortal.setRespectedGameType(type(uint32).max)` to invalidate games
+- Following outdated emergency procedures / misunderstanding version-specific behaviors
+
+This risk is heightened during emergency situations when quick action is required.
+
+### Risk Assessment
+
+- Impact: HIGH/CRITICAL
+  - Reasoning: Exact risk depends on the specific incident response mechanism impacted. For instance, if the chain operator calls `OptimismPortal.setRespectedGameType` assuming that it will invalidate all existing games, the operator may not understand that existing games can _still_ be used to finalize withdrawals (critical risk). Other failure modes within this category are generally MEDIUM/HIGH.
+- Likelihood: LOW
+  - Reasoning: Only applies during the first ~2 weeks after the upgrade when existing `FaultDisputeGame` contracts may still refer to the legacy `DelayedWETH` contract. Failure to hold funds would be easily noticable, only results in any impact if the `hold` operation occurs very close to the finalization time. Does not apply to any future upgrades.
+
+### Mitigations
+
+- Clear version documentation
+- Operator training on version differences
+- Version-specific runbooks
+- Retaining legacy `superchain-ops` tasks until further notice
+- Directing operators to reach out to OP Labs for assistance with any incident
+
+### Detection
+
+Dedicated detection apparatus not recommended. Very low likelihood of occuring in practice and OP Labs would almost certainly be made aware of any instance where these incident response capabilities are being used.
+
+### Recovery Path(s)
+
+- Immediate corrective actions (where applicable)
+- Operator retraining
+- Documentation updates
+
+### Action items
+
+- [ ] Create version-aware runbooks
+- [ ] Operator training sessions
+
+## FM3: Anchor state fails to progress within a time bound
+
+### Description
+
+The anchor state held within the `AnchorStateRegistry` could fail to progress and become too old. In particular, this could be possible if:
+
+- Guardian overuses the blacklisting/retirement mechanism to continuously block games from being used as the anchor state until the game is too old
+- Bug in `FaultDisputeGame` fails to update anchor state on game finalization
+- Bug in `AnchorStateRegistry` prevents games from updating the anchor state
+
+### Risk Assessment
+
+- Impact: HIGH
+  - Reasoning: Failure mode would block user withdrawals until resolved.
+- Likelihood: LOW/MEDIUM
+  - Reasoning: Unlikely to occur via bug or Guardian overuse in practice. Possible that this occurs if there is a bug with the `FaultDisputeGame`'s call to `setAnchorState` if no extra monitoring exists.
+
+#### Mitigations
+
+- Monitor for the age of the anchor state
+
+#### Detection
+
+- Monitor for the age of the anchor state
+
+#### Recovery Path(s)
+
+- Manual intervention to refresh the anchor state
```

Comment:
What does this look like, e.g. does it require a contract upgrade, or is there a setter method some privileged role can use? (Let's put the answer in the FMA text)

Reply:
Added in c19e884